### PR TITLE
Add AppLovin banner support

### PR DESCRIPTION
**Pull request by thomasmso**

### CODE REVIEW

**Comment by thomasmso** [comment]
I signed it!

**Comment by RedBrogdon** [comment]
Rerunning CLA check.

**Comment by RedBrogdon** [comment]
...and then running it again.

**Comment by thomasmso** [comment]
The adapter is ready for review. I've bumped the adapter version number as needed.

File: ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/ApplovinAdapter.java [hunk 1]
```diff
@@ -270,6 +283,178 @@ private AppLovinInterstitialAdDialog createInterstitial() {
...
     //endregion
 
+
+    //region - Banner Methods
+    @Override
+    public void requestBannerAd(final Context context, final MediationBannerListener mediationBannerListener, final Bundle serverParameters, final AdSize adSize, final MediationAdRequest mediationAdRequest, final Bundle mediationExtras)
+    {
+        // TODO: More elegant solution for retrieving SDK
+        mServerParameters = serverParameters;
+        mAdViewSdk = sdkInstance();
```
Comment (shankervennam) [inline ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/ApplovinAdapter.java]: The `sdkInstance()` uses [`mContext`](https://github.com/googleads/googleads-mobile-android-mediation/pull/30/files#diff-cfdfa427b089c2f44c47792796d8a7afL59) but it is not set. Please use the context passed to `requestBannerAd()`.

Comment (thomasmso) [reply ThirdPartyAdapters/applovin/applovin/src/main/java/com/applovin/mediation/ApplovinAdapter.java]: @shankervennam - I have made appropriate changes using dependency injection. Thank you for reviewing.

**Comment by thomasmso** [comment]
The adapter is ready for review.

**Pull request merged**